platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/HashSessionIdManager.java

```diff
@@ -62,10 +61,6 @@ public HashSessionIdManager() {
         super(RandomNumberGenerator.create());
     }
 
-    public HashSessionIdManager(Random random) {
```

Comment:
We need to leave this here. It isn't used by our own code, but Jetty could use this and if it isn't here, we'll get the wrong RNG.